src/pages/Swap/index.tsx

```diff
@@ -728,7 +725,6 @@ export function Swap({
               </ButtonError>
             </TraceEvent>
           )}
-          {Boolean(isExpertMode && swapError) && <SwapCallbackError error={swapError?.message} />}
```

Comment:
Should this callback error still be present if `swapError` is truthy?

Reply:
no, this was UI that only appeared in expert mode because those users didn't go through the normal flow. now all users should go through the ConfirmSwapModal flow which handles showing errors